ipatests/test_ipaclient/test_cli_ipa_not_configured.py

```diff
@@ -0,0 +1,39 @@
+import subprocess
+
+import pytest
+
+from ipapython.admintool import SERVER_NOT_CONFIGURED
+
+
+class TestIPANotConfigured:
+    """
+    Test class for CLI commands with ipa server not configured.
+    """
```

Comment:
The tests in the directory test_ipaclient are launched after installation of IPA server. Please see the doc for [ipa-docker-test-runner](https://github.com/freeipa/ipa-docker-test-runner): run-tests is calling install-server.

If you want your test to run on a non-installed server, you can either:
- move it to test_integration
- or enhance ipa-docker-test-runner with a new subcommand that would be used for tests without server installed. You can have a look at the [commit](https://github.com/freeipa/ipa-docker-test-runner/commit/621fad6624754c1e743fcec7a680ea256773f206) that added support for webui-unit subcommand. The new subcommand would be defined with @prerequisite(install_packages).